projects/core/src/cart/facade/multi-cart.service.ts

```diff
@@ -0,0 +1,242 @@
+import { Injectable } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { OrderEntry } from '../../model/order.model';
+import { CartActions } from '../store/actions/index';
+import { StateWithMultiCart } from '../store/multi-cart-state';
+import { MultiCartSelectors } from '../store/selectors/index';
+import { LoaderState } from '../../state/utils/loader/loader-state';
+import { Cart } from '../../model/cart.model';
+
+@Injectable()
+export class MultiCartService {
+  constructor(protected store: Store<StateWithMultiCart>) {}
+
+  /**
+   * Returns cart from store as an observable
+   *
+   * @param cartId
+   */
+  getCart(cartId: string): Observable<Cart> {
+    return this.store.pipe(
+      select(MultiCartSelectors.getCartSelectorFactory(cartId))
+    );
+  }
+
+  /**
+   * Returns cart entity from store (cart with loading, error, success flags) as an observable
+   *
+   * @param cartId
+   */
+  getCartEntity(cartId: string): Observable<LoaderState<Cart>> {
+    return this.store.pipe(
+      select(MultiCartSelectors.getCartEntitySelectorFactory(cartId))
+    );
+  }
+
+  /**
+   * Create or merge cart
+   *
+   * @param params Object with userId, oldCartId, toMergeCartGuid and extraData
+   */
+  createCart({
+    userId,
+    oldCartId,
+    toMergeCartGuid,
+    extraData,
+  }: {
+    userId: string;
+    oldCartId?: string;
+    toMergeCartGuid?: string;
+    extraData?: any;
+  }): Observable<LoaderState<Cart>> {
+    this.store.dispatch(
+      new CartActions.CreateCart({
+        extraData,
+        userId,
+        oldCartId,
+        toMergeCartGuid,
+      })
+    );
+    // to keep track of cart creation process we use cart with `fresh` id
+    // after creating cart we switch to entity with `code` or `guid`
+    // for loading/error state we need `fresh` cart entity
+    return this.getCartEntity('fresh');
```

Comment:
Bagatelle: For readability it would be good to extract `'fresh'` to some constant and also keep the /** comment */ there. Then anywhere you use this conts, you can eaisly find the purpose of this thing.